include/faabric/util/func.h

```diff
@@ -51,7 +51,7 @@ namespace faabric::util {
 
     bool isValidFunction(const faabric::Message &msg);
 
-    std::string funcToString(const faabric::Message &msg, bool includeId);
+    std::string funcToString(const faabric::Message &msg, bool includeId, bool includeSGX = false);
```

Comment:
Instead of adding another parameter we can use the `issgx` parameter of the messages